atomicapp/install.py

```diff
@@ -147,7 +147,7 @@ def install(self):
 
         printAnswerFile(json.dumps(answerContent))
         printStatus("Install Successful.")
-        return values
+        return None
```

Comment:
@goern run() also returns None.
The funny thing is i need None for the testcases to work.